src/main/java/uk/gov/hmcts/divorce/citizen/event/InviteApplicant2.java

```diff
@@ -40,44 +45,57 @@
     public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
 
         configBuilder
-            .event(CITIZEN_INVITE_APPLICANT_2)
+            .event(INVITE_APPLICANT_2)
             .forStateTransition(Draft, AwaitingApplicant2Response)
-            .name("Invite The Respondent")
-            .description("Send Application to the respondent for review")
-            .grant(CREATE_READ_UPDATE, CITIZEN)
+            .name("Invite Applicant 2")
+            .description("Invite Applicant 2")
+            .showCondition("applicant2SolicitorRepresented=\"Yes\" AND applicationType=\"jointApplication\"")
```

Comment:
take `applicant2SolicitorRepresented=\"Yes\"` off as it should work going from solicitor to citizen as well